src/api/utils.ts

```diff
@@ -1,5 +1,7 @@
 import axios, {AxiosError} from 'axios';
 import {RequestIdHeaderName} from './headers';
+import {LanguageAndText} from '../reference-data/types';
+import {Language} from '../translations';
```

Comment:
```suggestion
```

Probably don't need these imports?

Reply:
They are needed. Used for typing the function parameters.

Reply:
Didn't you move them?

Reply:
Aha. Just saw a comment on `utils.txt`, and didn't think it through 😅